src/lib/common/payload_uart.cpp

```diff
@@ -208,6 +331,10 @@ BaseType_t init(uint8_t task_priority) {
                                 2048, &PLUART::uart_handle, task_priority, NULL);
   configASSERT(rval == pdTRUE);
 
+  // Create the task to execute the _postTxFunction callback
```

Comment:
@victorsowa12 @towynlin @matt001k - Not sure if the PostTxCBTask should have the same priority we pass to `init`. It's gonna be relatively low priority and often include delays, so should probably be user task priority?

Reply:
To me it feels that if the callback might have random delays and potentially unknown behavior, it should be low priority to prevent any funky handling with other tasks.